tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
+)
+
+func BenchmarkStress25Environments0Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(25, 0, 0, 0, deployCommand, b)
+}
+func BenchmarkStress10Environments10Deploys0Services0LoadBalancers(b *testing.B) {
+	benchmarkStress(10, 10, 0, 0, deployCommand, b)
```

Comment:
[nit] helper functions for tests that take a `*testing.T` or `*testing.B` tend to be the first parameter.

Reply:
✅